crates/chain-config/Cargo.toml

```diff
@@ -25,13 +25,10 @@ serde_with = "1.11"
 tracing = "0.1"
 
 [dev-dependencies]
-fuel-core = { workspace = true }
```

Comment:
It causes cycle dependencies during publishing https://github.com/rust-lang/cargo/issues/4242.